app/models/profile.rb

```diff
@@ -11,8 +11,8 @@ class Profile < ActiveRecord::Base
 
   # Attachment
   has_attached_file :avatar,
-                    styles: {medium: '300x300>', thumb: '100x100>'},
-                    path: ':rails_root/storage/profile/:id/:style/:filename'
+    styles: {medium: '300x300>', thumb: '100x100>'},
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.